spec/board_spec.rb

```diff
@@ -0,0 +1,50 @@
+require './lib/board.rb'
+
+RSpec.describe Board do
+
+  let(:board){@board = Board.new}
+
+  describe '#display' do
+    output = 1..9
+    it 'displays the board ' do
+     expect(board.display).to eql(output)
+    end   
+  end 
+
+  describe '#position_taken?' do 
+    let(:board){@board = Board.new}
+    it 'checks if there is space on the board' do
+      expect(board.position_taken?(1)).to eql(false)
+    end
+  end
+
+  describe '#win?' do
+    let(:board){Board.new} 
+    it 'checks for win on board' do
+      expect(board.win?).to be false
+    end
+  end
+  
+  describe '#full?' do
+     let(:board){Board.new}
+     it 'checks if board is full' do
+      expect(board.full?).to eql(false)
+    end
+  end 
+
+  describe '#replace_var' do
+    let(:board){Board.new}
+    it 'replaces position with token on the board' do
+      expect(board.replace_var(1,'x')).to eql('x')

```

Comment:
Is this testing if the value have been really replaced? :thinking: 

Let's check the value of the board (`@board`) and the remaining value (`@remval`) too.

```suggestion
      expect(board.replace_var(1,'x')).to eql('x')
      expect(board.board).to eq [1, 'x', 3, 4, 5, 6, 7, 8, 9]
      expect(board.rem_nums).to eq [1, 3, 4, 5, 6, 7, 8, 9]
```

Reply:
done